Keycloak Admin Client in QuarkusTest*Callback

**Describe the bug**
First of all, I'm not really sure if it's a bug or me who does not understand how QuarkusTest Callbacks work.
I try to reset Keycloak Server environment between each of my test methods using **QuarkusTestAfterEachCallback** but as soon as I try to manipulate **org.keycloak.admin.client.Keycloak** instance inside **afterEach** Method I get an InvocationTargetException like the following :

`Caused by: java.lang.RuntimeException: java.lang.NoSuchMethodException: org.jboss.resteasy.client.jaxrs.internal.proxy.ProxyBuilderImpl.<init>(java.lang.Class, javax.ws.rs.client.WebTarget)
Caused by: java.lang.NoSuchMethodException: org.jboss.resteasy.client.jaxrs.internal.proxy.ProxyBuilderImpl.<init>(java.lang.Class, javax.ws.rs.client.WebTarget)`

It looks like a classloader issue but I'm out of idea how include this missing method.

**To Reproduce**

A small reproducer can be found there : https://github.com/Vinche59/test-callbacks

Steps to reproduce the behavior:
1. mvn test

--- a/test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java
+++ b/test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java
@@ -455,12 +455,12 @@ public class QuarkusTestExtension
         }
         if (!failedBoot) {
             popMockContext();
+            ClassLoader original = setCCL(runningQuarkusApplication.getClassLoader());
             for (Object afterEachCallback : afterEachCallbacks) {
                 Map.Entry<Class<?>, ?> tuple = createQuarkusTestMethodContextTuple(context);
                 afterEachCallback.getClass().getMethod("afterEach", tuple.getKey())
                         .invoke(afterEachCallback, tuple.getValue());
             }
-            ClassLoader original = setCCL(runningQuarkusApplication.getClassLoader());
             try {
                 runningQuarkusApplication.getClassLoader().loadClass(RestAssuredURLManager.class.getName())
                         .getDeclaredMethod("clearURL").invoke(null);

